eventmesh-common/src/main/java/org/apache/eventmesh/common/protocol/http/common/ClientRetCode.java

```diff
@@ -21,6 +21,7 @@
 
     /**
      * 这个RETRY的意思是 客户端发现投递的消息它没有监听时, 告诉EventMesh 发往下一个, 重试几次以实现灰度 , 预留
```

Comment:
We may also need to remove this Chinese comment

Reply:
Got it.